### PR TITLE
 Makefile: suitable "default" target

### PR DESCRIPTION
**Pull request by q384566678**
Signed-off-by: zhouhao <zhouhao@cn.fujitsu.com>

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -10,7 +10,7 @@ TOOLS := \
 	oci-unpack
 MAN := $(TOOLS:%=%.1)
 
-default: help
+default: check-license lint test
```
Comment (cyphar) [inline Makefile]: Why is this the most obvious default target? Why not `all` or `tools`?

Comment (q384566678) [inline Makefile]: updated

**Comment by q384566678** [comment]
@xiekeyang @coolljt0725 @stevvooe @cyphar PTAL

**Comment by cyphar** [comment]
LGTM

**Comment by stevvooe** [comment]
LGTM

**Pull request merged**